lighthouse-core/audits/resource-summary.js

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license Copyright 2019 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const NetworkRecords = require('../computed/network-records.js');
+const ComputedResourceSummary = require('../computed/resource-summary.js');
+const i18n = require('../lib/i18n/i18n.js');
+const MainResource = require('../computed/main-resource.js');
+
+const UIStrings = {
+  /** Imperative title of a Lighthouse audit that tells the user to minimize the size and quantity of resources used to load the page. */
+  title: 'Keep request counts and transfer sizes small',
```

Comment:
WDYT about `s/small/low`?

Reply:
🚲🏡: Semantics edition
I think I have a slight preference for how it currently is because to me, low:requestCount & small:transferSizes, and "small" comes after "transfer sizes".
How do you feel about "Keep request counts low and transfer sizes small"?

Reply:
hm interesting, both feel like "low" to me because the transfer size is something I can count despite the misleading "size" term in there, it's the payloads/resources that are kept small in my crazy 🧠

I can see the argument for small sizes though given "size" being generally unquantifiable and it might feel weird to folks unfamiliar with what transfer size is measured in (maybe a separate issue?), so that verbiage works for me!